com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1330,6 +1347,10 @@ private void ConfigureSimulator()
                 , mode: ApplyMode.AllPackets
 #endif
             );
+
+#if UTP_TRANSPORT_2_0_ABOVE
```

Comment:
Should this PR not introduce this compiler directive? Or is there another tracking ticket for when it will be introduced?

Reply:
This compile directive is already there through the existence of UTP 2. It was added by NGO team to support UTP 2.

Reply:
Ah okay, thanks for confirming